src/store/wizardSlice.ts

```diff
@@ -1560,8 +1566,8 @@ export const {
   setUserPasswordByIndex,
   setUserSshKeyByIndex,
   setUserAdministratorByIndex,
-  addUserGroupByIndex,
-  removeUserGroupByIndex,
+  assignGroupToUserByIndex,
```

Comment:
Nitpick: I think we should maybe stick with `add` rather than `assign`. Just for consistency.

Since we're tidying this up though, I think `ByIndex` is also kind of ambiguous. Is it the user index or the group index? Maybe we could do something like `addGroupToIndexedUser` or maybe more verbose (and clearer) `addGroupToUserByUserIndex`.